test/Kestrel.Transport.BindTests/HandleInheritanceTest.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.FunctionalTests
+{
+    [OSSkipCondition(OperatingSystems.MacOSX)] // Not applicable, and vi doesn't work there.
+    public class HandleInheritanceTest : TestApplicationErrorLoggerLoggedTest
+    {
+        [ConditionalFact]
+        public async Task SpawnChildProcess_DoesNotInheritListenHandle()
```

Comment:
I tried running this test on my Windows 10 machine and it passed without your changes.